schema/filter_artworks.js

```diff
@@ -145,6 +145,10 @@ function filterArtworks(primaryKey) {
         delete gravityOptions.medium;
       }
 
+      if (accessToken) {
+        return gravity.with(accessToken)('filter/artworks', gravityOptions);
+      }
+
       return gravity('filter/artworks', gravityOptions);
```

Comment:
Yea, it'll bypass caching completely if you use `with` (whether you pass a token in or not), since that uses the authenticated loader (which does no caching).

In that case we should go this for now until we fix that?